### PR TITLE
Update documentation for strict host tags

### PR DESCRIPTION
**Pull request by vishesh92**
Related PR: https://github.com/apache/cloudstack/pull/9017

----
📚 Documentation preview 📚: https://cloudstack-documentation--398.org.readthedocs.build/en/398/

### CODE REVIEW

**Comment by vishesh92** [comment]
@blueorangutan docbuild

File: source/adminguide/host_and_storage_tags.rst [hunk 1]
```diff
@@ -49,6 +49,37 @@ To explain the behavior of host tags, some examples will be demonstrated with tw
     * Offering: h3
     None of the hosts have compatible tags and it will not be possible to deploy a VM with the offering. However, CloudStack ignores this behavior when a host is manually selected.
 
+.. _strict-host-tags:
+Strict Host Tags
+-----------------
+During certain operations, like changing compute offering, and starting or live
+migrating an instance to a specific host, CloudStack will ignore the host tags,
+which can lead to issues where a instance starts on an undesired host.
```
Comment (andrijapanicsb) [inline source/adminguide/host_and_storage_tags.rst:57]: We should (?) reword this to explain that "ignoring tags" is mostly (or only?) done when Admin does some operations (and this is by design/desired in most cases) - however we are adding an add-on feature to force tags checks by this global setting.

File: source/adminguide/host_and_storage_tags.rst [hunk 2]
```diff
@@ -49,6 +49,37 @@ To explain the behavior of host tags, some examples will be demonstrated with tw
...
+
+To avoid a mismatch of tags, tags which must always match during any operation
+can be defined in `vm.strict.host.tags`. If the tags which doesn't match are
```
Comment (andrijapanicsb) [inline source/adminguide/host_and_storage_tags.rst:60]: explain this is global configuration, it doesn't say it's a global config 'vm.strict.host.tags'....

**Comment by vishesh92** [comment]
@andrijapanicsb I have updated the documentation. Please review again.

**Comment by andrijapanicsb** [comment]
I think it makes more sense now, yes. Thanks Vishesh.

**Review by vladimirpetrov — APPROVED** [review]
LGTM
